### PR TITLE
feat: create escrow page

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

File: app/js/pages/Escrow/index.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@import "../../../css//variable-overrides.scss";
+
+.escrow {
+  .escrow--split {
+    background-color: $secondary;
```
Comment (jrainville) [inline app/js/pages/Escrow/index.scss]: There is `.bg-secondary` that you can use

File: app/js/pages/Profile/index.jsx [hunk 2]
```diff
@@ -29,7 +31,16 @@ class Profile extends Component {
         <h3 className="mt-3">{profile.location}</h3>
         <Map coords={{latitude: 45.492611, longitude: -73.617959}} markerOnly={true}/>
         <p className="text-muted mt-2">Saalestraße 39A, 12055 Berlin</p>
-        <Offers offers={profile.offers} onClick={this.offerClick}/>
+        <Row>
+          <Col xs="12" className="mt-2">
```
Comment (jrainville) [inline app/js/pages/Profile/index.jsx]: IMO, doing a row just for one col doesn't do much, it's the same as a normal div at this point. Am I missing something?

File: app/js/ui/RoundedIcon/index.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+@import "../../../css/variable-overrides.scss";
+
+.rounded-icon {
+  padding: 10px;
+}
+
+.rounded-icon__grey {
+  background-color: #939BA1;
```
Comment (jrainville) [inline app/js/ui/RoundedIcon/index.scss]: If the color is not in the variables, you can add it. We will probably reuse it.

File: app/js/pages/Profile/index.jsx (hunk 2 above)
Comment (alaibe) [reply app/js/pages/Profile/index.jsx]: Yes, Row are removing the padding from the container, so the background is full width

Comment (jrainville) [reply app/js/pages/Profile/index.jsx]: Ah ok, understood

File: app/js/pages/Escrow/index.scss (hunk 1 above)
Comment (alaibe) [reply app/js/pages/Escrow/index.scss]: updated

File: app/js/ui/RoundedIcon/index.scss (hunk 3 above)
Comment (alaibe) [reply app/js/ui/RoundedIcon/index.scss]: updated, it was but I missed it!!

**Pull request merged**